website/content/en/docs/concepts/services.md

```diff
@@ -1,26 +1,204 @@
+ in a peer by defining an `Import` CRD instance in the destination cluster[^KEP-1645].
+ Thus, service sharing is an explicit operation. Services are not automatically
+ shared by peers in the fabric. Note that the exporting cluster must be
+ [configured as a peer]({{% ref "peers#add-or-remove-peers" %}}) of the importing
+ cluster.
+
+A service is shared using a logical name. The logical name does not have to match
+ the actual Kubernetes Service name in the exporting cluster. Exporting a service
+ does not expose cluster Pods or their IP addresses to the importing clusters.
+ Any load balancing and scaling decisions are kept local in the exporting cluster.
+ This reduces the amount, frequency and sensitivity of information shared between
+ clusters. Similarly, the imported service can have any arbitrary name in the
+ destination cluster, allowing independent choice of naming.
+
+Orchestration of service sharing is the responsibility of users wishing to
+ export or import it, and any relevant information (e.g, the exported service
+ name and namespace) must be communicated out of band. In the future, this could
+ be done by a centralized management plane.
+
+<!-- TODO: image showing export/import (from >2 clusters?) -->
+
+<!-- 
+ TODO centralized management may apply simplification of sharing via policies
+  (e.g., auto-expose)
+  can also simplify some via clusterlink cli (e.g., allow-all policy default)
+  Ask for inputs through the documentation?
+  Put all of the above in footnote?
 -->
 
-## What is it
+## Prerequisites
+
+The following assume that you have `kubectl` access to two or more clusters where ClusterLink
+ has already been [deployed and configured]({{% ref "users#setup" %}}).
+
+### Exporting a service
+
+In order to make a service potentially accessible by other clusters, it must be
+ explicitly configured for remote access via ClusterLink. Exporting is
+ accomplished by creating an `Export` CRD instance in the **same** namespace
+ as the service being exposed. The CRD instance acts as a marker for enabling
+ remote access to the service via ClusterLink.
+
+{{% expand summary="Export Custom Resource" %}}
+
+```go
+type Export struct {
+    metav1.TypeMeta   `json:",inline"`
+    metav1.ObjectMeta `json:"metadata,omitempty"`
+
+    Spec ExportSpec     `json:"spec,omitempty"`
+    Status ExportStatus `json:"status,omitempty"`
+}
+
+type ExportSpec struct {
+    Host string `json:"host,omitempty"`
+    Port uint16 `json:"port,omitempty"`
+}
+
+type ExportStatus struct {
+    Conditions []metav1.Condition `json:"conditions,omitempty"`
+}
+```
+
+{{% /expand %}}
+
+The ExportSpec defines the following fields:
+
+- **Host** (string, optional): the name of the service being exported. If empty,
+ the export shall refer to a Kubernetes Service with the same name as the instance's
+ `metadata.name`. Currently, only TCP based services can be exported[^UDP]. It is an
```

Comment:
actually is `metadata.name.`metadata.nnamespace.svc.cluster.local

Reply:
this refers to the name inside the same namespace.
I think it should be clear since Service is always namespaced.